<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<title>22 References | A gentle introduction to R for biologists</title>
<meta name="author" content="Kittinun Leetanaporn">
<meta name="description" content="Burger, T. (2018). Gentle Introduction to the Statistical Foundations of False Discovery Rate in Quantitative Proteomics. Journal of Proteome Research, 17(1), 12–22....">
<meta name="generator" content="bookdown 0.36 with bs4_book()">
<meta property="og:title" content="22 References | A gentle introduction to R for biologists">
<meta property="og:type" content="book">
<meta property="og:description" content="Burger, T. (2018). Gentle Introduction to the Statistical Foundations of False Discovery Rate in Quantitative Proteomics. Journal of Proteome Research, 17(1), 12–22....">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="22 References | A gentle introduction to R for biologists">
<meta name="twitter:description" content="Burger, T. (2018). Gentle Introduction to the Statistical Foundations of False Discovery Rate in Quantitative Proteomics. Journal of Proteome Research, 17(1), 12–22....">
<!-- JS --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://kit.fontawesome.com/6ecbd6c532.js" crossorigin="anonymous"></script><script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<link href="libs/bootstrap-4.6.0/bootstrap.min.css" rel="stylesheet">
<script src="libs/bootstrap-4.6.0/bootstrap.bundle.min.js"></script><script src="libs/bs3compat-0.5.1/transition.js"></script><script src="libs/bs3compat-0.5.1/tabs.js"></script><script src="libs/bs3compat-0.5.1/bs3compat.js"></script><link href="libs/bs4_book-1.0.0/bs4_book.css" rel="stylesheet">
<script src="libs/bs4_book-1.0.0/bs4_book.js"></script><link href="libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet">
<script src="libs/pagedtable-1.1/js/pagedtable.js"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- CSS --><style type="text/css">
    
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  </style>
<style type="text/css">
    /* Used with Pandoc 2.11+ new --citeproc when CSL is used */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
        }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
</head>
<body data-spy="scroll" data-target="#toc">

<div class="container-fluid">
<div class="row">
  <header class="col-sm-12 col-lg-3 sidebar sidebar-book"><a class="sr-only sr-only-focusable" href="#content">Skip to main content</a>

    <div class="d-flex align-items-start justify-content-between">
      <h1>
        <a href="index.html" title="">A gentle introduction to R for biologists</a>
      </h1>
      <button class="btn btn-outline-primary d-lg-none ml-2 mt-1" type="button" data-toggle="collapse" data-target="#main-nav" aria-expanded="true" aria-controls="main-nav"><i class="fas fa-bars"></i><span class="sr-only">Show table of contents</span></button>
    </div>

    <div id="main-nav" class="collapse-lg">
      <form role="search">
        <input id="search" class="form-control" type="search" placeholder="Search" aria-label="Search">
</form>

      <nav aria-label="Table of contents"><h2>Table of contents</h2>
        <ul class="book-toc list-unstyled">
<li><a class="" href="index.html">Welcome!</a></li>
<li class="book-part">Basic R</li>
<li><a class="" href="basic-r.html"><span class="header-section-number">1</span> Basic R</a></li>
<li><a class="" href="r-function.html"><span class="header-section-number">2</span> R function</a></li>
<li><a class="" href="loop.html"><span class="header-section-number">3</span> Loop</a></li>
<li class="book-part">Tidyverse</li>
<li><a class="" href="what-is-tidyverse.html"><span class="header-section-number">4</span> What is tidyverse</a></li>
<li><a class="" href="importing-data.html"><span class="header-section-number">5</span> Importing data</a></li>
<li><a class="" href="data-wrangling.html"><span class="header-section-number">6</span> Data wrangling</a></li>
<li><a class="" href="data-visualization-with-ggplot2.html"><span class="header-section-number">7</span> Data visualization with ggplot2</a></li>
<li><a class="" href="GSE63514.html"><span class="header-section-number">8</span> Example: Microarray data visualization</a></li>
<li class="book-part">Statistical analysis</li>
<li><a class="" href="hypothesis-testing.html"><span class="header-section-number">9</span> Hypothesis testing</a></li>
<li><a class="" href="parametric-test.html"><span class="header-section-number">10</span> Parametric test</a></li>
<li><a class="" href="non-parametric-test.html"><span class="header-section-number">11</span> Non-parametric test</a></li>
<li><a class="" href="false-discovery.html"><span class="header-section-number">12</span> False discovery</a></li>
<li><a class="" href="regression-model.html"><span class="header-section-number">13</span> Regression model</a></li>
<li><a class="" href="survival-analysis.html"><span class="header-section-number">14</span> Survival analysis</a></li>
<li><a class="" href="principal-component-analysis.html"><span class="header-section-number">15</span> Principal component analysis</a></li>
<li class="book-part">Bioconductor</li>
<li><a class="" href="bioconductor.html"><span class="header-section-number">16</span> Bioconductor</a></li>
<li><a class="" href="limma.html"><span class="header-section-number">17</span> Limma</a></li>
<li><a class="" href="edger.html"><span class="header-section-number">18</span> edgeR</a></li>
<li><a class="" href="pathway-analysis.html"><span class="header-section-number">19</span> Pathway analysis</a></li>
<li class="book-part">Supplement</li>
<li><a class="" href="distribution.html"><span class="header-section-number">20</span> Distribution</a></li>
<li class="book-part">Information</li>
<li><a class="" href="session-info.html"><span class="header-section-number">21</span> Session info</a></li>
<li><a class="active" href="references.html"><span class="header-section-number">22</span> References</a></li>
</ul>

        <div class="book-extra">
          
        </div>
      </nav>
</div>
  </header><main class="col-sm-12 col-md-9 col-lg-7" id="content"><div id="references" class="section level1" number="22">
<h1>
<span class="header-section-number">22</span> References<a class="anchor" aria-label="anchor" href="#references"><i class="fas fa-link"></i></a>
</h1>

<div id="refs" class="references csl-bib-body hanging-indent" line-spacing="2">
<div id="ref-burger2018" class="csl-entry">
Burger, T. (2018). Gentle Introduction to the Statistical Foundations of False Discovery Rate in Quantitative Proteomics. <em>Journal of Proteome Research</em>, <em>17</em>(1), 12–22. <a href="https://doi.org/10.1021/acs.jproteome.7b00170">https://doi.org/10.1021/acs.jproteome.7b00170</a>
</div>
<div id="ref-edgeR2016" class="csl-entry">
Chen, Y., Lun, A. A. T., &amp; Smyth, G. K. (2016). From reads to genes to pathways: Differential expression analysis of RNA-seq experiments using rsubread and the edgeR quasi-likelihood pipeline. <em>F1000Research</em>, <em>5</em>, 1438. <a href="https://doi.org/10.12688/f1000research.8987.2">https://doi.org/10.12688/f1000research.8987.2</a>
</div>
<div id="ref-R-edgeR" class="csl-entry">
Chen, Y., Lun, A. T., McCarthy, D. J., Ritchie, M. E., Phipson, B., Hu, Y., … Smyth, G. K. (2023). <em>edgeR: Empirical analysis of digital gene expression data in r</em>. <a href="https://doi.org/10.18129/B9.bioc.edgeR">https://doi.org/10.18129/B9.bioc.edgeR</a>
</div>
<div id="ref-chicco2022" class="csl-entry">
Chicco, D., &amp; Agapito, G. (2022). Nine quick tips for pathway enrichment analysis. <em>PLOS Computational Biology</em>, <em>18</em>(8), e1010348. <a href="https://doi.org/10.1371/journal.pcbi.1010348">https://doi.org/10.1371/journal.pcbi.1010348</a>
</div>
<div id="ref-denboon2015" class="csl-entry">
Den Boon, J. A., Pyeon, D., Wang, S. S., Horswill, M., Schiffman, M., Sherman, M., … Ahlquist, P. (2015). Molecular transitions from papillomavirus infection to cervical precancer and cancer: Role of stromal estrogen receptor signaling. <em>Proceedings of the National Academy of Sciences</em>, <em>112</em>(25). <a href="https://doi.org/10.1073/pnas.1509322112">https://doi.org/10.1073/pnas.1509322112</a>
</div>
<div id="ref-R-msigdbr" class="csl-entry">
Dolgalev, I. (2022). <em>Msigdbr: MSigDB gene sets for multiple organisms in a tidy data format</em>. Retrieved from <a href="https://igordot.github.io/msigdbr/">https://igordot.github.io/msigdbr/</a>
</div>
<div id="ref-gillespie2022" class="csl-entry">
Gillespie, M., Jassal, B., Stephan, R., Milacic, M., Rothfels, K., Senff-Ribeiro, A., … D’Eustachio, P. (2022). The reactome pathway knowledgebase 2022. <em>Nucleic Acids Research</em>, <em>50</em>(D1), D687–D692. <a href="https://doi.org/10.1093/nar/gkab1028">https://doi.org/10.1093/nar/gkab1028</a>
</div>
<div id="ref-j.h." class="csl-entry">
J. H., M. (n.d.). <em>Handbook of biological statistics</em> (3rd ed.). Baltimore, Maryland: Sparky House Publishing. Retrieved from <a href="https://www.biostathandbook.com/">https://www.biostathandbook.com/</a>
</div>
<div id="ref-kanehisa2016" class="csl-entry">
Kanehisa, M., Sato, Y., Kawashima, M., Furumichi, M., &amp; Tanabe, M. (2016). KEGG as a reference resource for gene and protein annotation. <em>Nucleic Acids Research</em>, <em>44</em>(D1), D457–D462. <a href="https://doi.org/10.1093/nar/gkv1070">https://doi.org/10.1093/nar/gkv1070</a>
</div>
<div id="ref-liberzon2011" class="csl-entry">
Liberzon, A., Subramanian, A., Pinchback, R., Thorvaldsdóttir, H., Tamayo, P., &amp; Mesirov, J. P. (2011). Molecular signatures database (MSigDB) 3.0. <em>Bioinformatics</em>, <em>27</em>(12), 1739–1740. <a href="https://doi.org/10.1093/bioinformatics/btr260">https://doi.org/10.1093/bioinformatics/btr260</a>
</div>
<div id="ref-edgeR2012" class="csl-entry">
McCarthy, D. J., Chen, Y., &amp; Smyth, G. K. (2012). Differential expression analysis of multifactor RNA-seq experiments with respect to biological variation. <em>Nucleic Acids Research</em>, <em>40</em>(10), 4288–4297. <a href="https://doi.org/10.1093/nar/gks042">https://doi.org/10.1093/nar/gks042</a>
</div>
<div id="ref-R-base" class="csl-entry">
R Core Team. (2023). <em>R: A language and environment for statistical computing</em>. Vienna, Austria: R Foundation for Statistical Computing. Retrieved from <a href="https://www.R-project.org/">https://www.R-project.org/</a>
</div>
<div id="ref-R-MASS" class="csl-entry">
Ripley, B. (2023). <em>MASS: Support functions and datasets for venables and ripley’s MASS</em>. Retrieved from <a href="http://www.stats.ox.ac.uk/pub/MASS4/">http://www.stats.ox.ac.uk/pub/MASS4/</a>
</div>
<div id="ref-limma2015" class="csl-entry">
Ritchie, M. E., Phipson, B., Wu, D., Hu, Y., Law, C. W., Shi, W., &amp; Smyth, G. K. (2015). <span class="nocase">limma</span> powers differential expression analyses for <span>RNA</span>-sequencing and microarray studies. <em>Nucleic Acids Research</em>, <em>43</em>(7), e47. <a href="https://doi.org/10.1093/nar/gkv007">https://doi.org/10.1093/nar/gkv007</a>
</div>
<div id="ref-edgeR2010" class="csl-entry">
Robinson, M. D., McCarthy, D. J., &amp; Smyth, G. K. (2010). edgeR: A bioconductor package for differential expression analysis of digital gene expression data. <em>Bioinformatics</em>, <em>26</em>(1), 139–140. <a href="https://doi.org/10.1093/bioinformatics/btp616">https://doi.org/10.1093/bioinformatics/btp616</a>
</div>
<div id="ref-smith1987" class="csl-entry">
Smith, L. I. (1987). <em>A tutorial on Principal Components Analysis</em>. New York: Wiley.
</div>
<div id="ref-R-limma" class="csl-entry">
Smyth, G., Hu, Y., Ritchie, M., Silver, J., Wettenhall, J., McCarthy, D., … Choi, D. (2023). <em>Limma: Linear models for microarray data</em>. <a href="https://doi.org/10.18129/B9.bioc.limma">https://doi.org/10.18129/B9.bioc.limma</a>
</div>
<div id="ref-sohil2022" class="csl-entry">
Sohil, F., Sohali, M. U., &amp; Shabbir, J. (2022). <em>An introduction to statistical learning with applications in R</em> (Vol. 6). Retrieved from <a href="https://www.tandfonline.com/doi/full/10.1080/24754269.2021.1980261">https://www.tandfonline.com/doi/full/10.1080/24754269.2021.1980261</a>
</div>
<div id="ref-subramanian2005" class="csl-entry">
Subramanian, A., Tamayo, P., Mootha, V. K., Mukherjee, S., Ebert, B. L., Gillette, M. A., … Mesirov, J. P. (2005). Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. <em>Proceedings of the National Academy of Sciences</em>, <em>102</em>(43), 15545–15550. <a href="https://doi.org/10.1073/pnas.0506580102">https://doi.org/10.1073/pnas.0506580102</a>
</div>
<div id="ref-survival-book" class="csl-entry">
Terry M. Therneau, &amp; Patricia M. Grambsch. (2000). <em>Modeling survival data: Extending the <span>C</span>ox model</em>. New York: Springer.
</div>
<div id="ref-R-survival" class="csl-entry">
Therneau, T. M. (2023). <em>Survival: Survival analysis</em>. Retrieved from <a href="https://github.com/therneau/survival">https://github.com/therneau/survival</a>
</div>
<div id="ref-thomas2022" class="csl-entry">
Thomas, P. D., Ebert, D., Muruganujan, A., Mushayahama, T., Albou, L.-P., &amp; Mi, H. (2022). PANTHER: Making genome<span>-</span>scale phylogenetics accessible to all. <em>Protein Science : A Publication of the Protein Society</em>, <em>31</em>(1), 8. <a href="https://doi.org/10.1002/pro.4218">https://doi.org/10.1002/pro.4218</a>
</div>
<div id="ref-MASS2002" class="csl-entry">
Venables, W. N., &amp; Ripley, B. D. (2002). <em>Modern applied statistics with s</em> (Fourth). New York: Springer. Retrieved from <a href="https://www.stats.ox.ac.uk/pub/MASS4/">https://www.stats.ox.ac.uk/pub/MASS4/</a>
</div>
<div id="ref-walck" class="csl-entry">
Walck, C. (n.d.). <em>Handbook on statistical distributions for experimentalists</em>. Fysikum University of Stockholm, Stockholm: Particle Physics Group. Retrieved from <a href="https://www.stat.rice.edu/~dobelman/textfiles/DistributionsHandbook.pdf">https://www.stat.rice.edu/~dobelman/textfiles/DistributionsHandbook.pdf</a>
</div>
<div id="ref-ggplot2" class="csl-entry">
Wickham, H. (2016). <em>ggplot2: Elegant graphics for data analysis</em>. Springer-Verlag New York. Retrieved from <a href="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a>
</div>
<div id="ref-R-tidyverse" class="csl-entry">
Wickham, H. (2023). <em>Tidyverse: Easily install and load the tidyverse</em>. Retrieved from <a href="https://tidyverse.tidyverse.org">https://tidyverse.tidyverse.org</a>
</div>
<div id="ref-tidyverse2019" class="csl-entry">
Wickham, H., Averick, M., Bryan, J., Chang, W., McGowan, L. D., François, R., … Yutani, H. (2019). Welcome to the <span class="nocase">tidyverse</span>. <em>Journal of Open Source Software</em>, <em>4</em>(43), 1686. <a href="https://doi.org/10.21105/joss.01686">https://doi.org/10.21105/joss.01686</a>
</div>
<div id="ref-clusterProfiler2021" class="csl-entry">
Wu, T., Hu, E., Xu, S., Chen, M., Guo, P., Dai, Z., … Yu, G. (2021). clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. <em>The Innovation</em>, <em>2</em>(3), 100141. <a href="https://doi.org/10.1016/j.xinn.2021.100141">https://doi.org/10.1016/j.xinn.2021.100141</a>
</div>
<div id="ref-R-clusterProfiler" class="csl-entry">
Yu, G. (2023). <em>clusterProfiler: A universal enrichment tool for interpreting omics data</em>. <a href="https://doi.org/10.18129/B9.bioc.clusterProfiler">https://doi.org/10.18129/B9.bioc.clusterProfiler</a>
</div>
<div id="ref-clusterProfiler2012" class="csl-entry">
Yu, G., Wang, L.-G., Han, Y., &amp; He, Q.-Y. (2012). clusterProfiler: An r package for comparing biological themes among gene clusters. <em>OMICS: A Journal of Integrative Biology</em>, <em>16</em>(5), 284–287. <a href="https://doi.org/10.1089/omi.2011.0118">https://doi.org/10.1089/omi.2011.0118</a>
</div>
</div>
</div>
  <div class="chapter-nav">
<div class="prev"><a href="session-info.html"><span class="header-section-number">21</span> Session info</a></div>
<div class="empty"></div>
</div></main><div class="col-md-3 col-lg-2 d-none d-md-block sidebar sidebar-chapter">
    <nav id="toc" data-toggle="toc" aria-label="On this page"><h2>On this page</h2>
      <ul class="nav navbar-nav"><li><a class="nav-link" href="#references"><span class="header-section-number">22</span> References</a></li></ul>

      <div class="book-extra">
        <ul class="list-unstyled">
          
        </ul>
</div>
    </nav>
</div>

</div>
</div> <!-- .container -->

<footer class="bg-primary text-light mt-5"><div class="container"><div class="row">

  <div class="col-12 col-md-6 mt-3">
    <p>"<strong>A gentle introduction to R for biologists</strong>" was written by Kittinun Leetanaporn. It was last built on 2023-11-01.</p>
  </div>

  <div class="col-12 col-md-6 mt-3">
    <p>This book was built by the <a class="text-light" href="https://bookdown.org">bookdown</a> R package.</p>
  </div>

</div></div>
</footer><!-- dynamically load mathjax for compatibility with self-contained --><script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script><script type="text/x-mathjax-config">const popovers = document.querySelectorAll('a.footnote-ref[data-toggle="popover"]');
for (let popover of popovers) {
  const div = document.createElement('div');
  div.setAttribute('style', 'position: absolute; top: 0, left:0; width:0, height:0, overflow: hidden; visibility: hidden;');
  div.innerHTML = popover.getAttribute('data-content');

  var has_math = div.querySelector("span.math");
  if (has_math) {
    document.body.appendChild(div);
    MathJax.Hub.Queue(["Typeset", MathJax.Hub, div]);
    MathJax.Hub.Queue(function() {
      popover.setAttribute('data-content', div.innerHTML);
      document.body.removeChild(div);
    })
  }
}
</script>
</body>
</html>
